reflex.cabal

```diff
@@ -1,5 +1,5 @@
 Name: reflex
-Version: 0.5.0.1
```

Comment:
I think this needs to be 0.6, since deprecating `FunctorMaybe` is a breaking change.